install/mysql/glpi-9.2-empty.sql

```diff
@@ -2047,6 +2047,86 @@ CREATE TABLE `glpi_deviceprocessors` (
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 
 
+### Dump table glpi_devicesimcards
+
+DROP TABLE IF EXISTS `glpi_devicesimcards`;
+CREATE TABLE IF NOT EXISTS `glpi_devicesimcards` (
+   `id` int(11) NOT NULL AUTO_INCREMENT,
+   `designation` varchar(255) NOT NULL DEFAULT '',
```

Comment:
Why not using name as in other devices? This sounds more work to use something else..